core-bundle/contao/classes/DataContainer.php

```diff
@@ -1042,7 +1042,16 @@ public function initPicker(PickerInterface $picker)
 	{
 		$provider = $picker->getCurrentProvider();
 
-		if (!$provider instanceof DcaPickerProviderInterface || $provider->getDcaTable($picker->getConfig()) != $this->strTable)
+		if (!$provider instanceof DcaPickerProviderInterface || $provider->getDcaTable($picker->getConfig()) != $this->strTable || Input::get('act') == 'select')
```

Comment:
The early return here prevents the tree from showing the selected nodes.

### Regular mode

<img width="902" alt="" src="https://github.com/user-attachments/assets/8141761b-d503-4aac-b717-301b35e0a97b">

### Clicked on "edit multiple"

<img width="901" alt="" src="https://github.com/user-attachments/assets/05d391f0-6d11-4290-8391-df94a43a4a9c">

Reply:
I think that is intentional. We now display _select all_ **OR** _paste_ **OR** _the picker_. You won't loose that value if you paste first.

Reply:
fixed in 8ec639c